test/configCases/side-effects/dead-control-flow/index.js

```diff
@@ -0,0 +1,19 @@
+it("should not include unused assets", (done) => {
+	if (true) {
+		done();
+		return;
+	}
+
+	require("should not require");
+});
+
+it("should not include unused assets", (done) => {
+	try {
+		if (true) {
```

Comment:
Make sure you have a test case for `try/finally`:

```js
try {
  return;
} finally {
   // This code *will* execute after the 'return'
}
```

Reply:
@JSerFeng can you add a test for this please?

Reply:
@dmichon-msft we have other PR for such stuff(link above), we need to merge it firstly, I will take care about it

Reply:
But they can union in some places, that’s is why we need to make it step by step